internal/commands/ip_access/inputs.go

```diff
@@ -0,0 +1,21 @@
+package ipaccess
+
+const (
+	flagIP            = "ip"
+	flagIPUsageCreate = "the allowed IP address"
+	flagIPUsageUpdate = "the current allowed IP address"
+	flagIPUsageDelete = "delete the allowed IP address"
+
+	flagNewIP            = "new-ip"
+	flagNewIPUsageUpdate = "the new allowed IP address"
+
+	flagComment            = "comment"
+	flagCommentUsageCreate = "the comment of the allowd IP address"
+	flagCommentUsageUpdate = "the new comment of the allowed IP"
+
+	flagUseCurrent            = "use-current"
+	flagUseCurrentUsageCreate = "use current IP address"
+
+	flagAllowAll            = "allow-all"
```

Comment:
one note on flags: the codebase still hasn't really landed on a solid pattern for these yet.  so first and foremost, what you have here is fine, don't sweat the organization (or re-use) of flags all that much.

I can see the point of some shared "names"/concepts for flags, but its rarely the case a usage string wants to be exactly the same across commands. just food for thought, in case you have any ideas here